### DISCUSSION TITLE
Get color area

### DISCUSSION THREAD

**Discussion by jueqing1015** [dlemstra/Magick.NET#725] [Help]
As shown in the figure below, the background in the figure is white and there is a black area. Is there a way to get the position of the black area?The black area is just a rectangle
It is a bit like the existing Trim() function, but the Trim() function cannot get the cropped information
![abc](https://user-images.githubusercontent.com/65677359/91993632-32a0c480-ed68-11ea-9ded-34a877fb6239.jpg)

**Comment by dlemstra** [comment]
You can get the information from the `image.BoundingBox` method. You might need to set `image.ColorFuzz` to make sure you don't get some of the gray pixels close to the black area.

**Reply by jueqing1015** [reply]
Hi, I tried it, it seems a bit lacking.
As shown below:
I want to use the mouse to draw a rough area on the image, and then get the coordinates of the text part according to what I want, which is similar to the TRIM function, because I tried it with TRIM, and it can be cropped after the two values.
What I want to do now is to draw a rough area on a big picture, and then shrink according to the area. In the picture, red is the rough area and blue is the last area.
![ewf](https://user-images.githubusercontent.com/65677359/92389452-ae758500-f14b-11ea-9736-8c119a4ec0e2.jpg)

**Reply by dlemstra** [reply]
I don't understand why the BoundingBox property does not give you the necessary information.

**Reply by jueqing1015** [reply]
The information is there, but now I want to draw a rough rectangle on the image first, and then perform BoundingBox based on the drawn rectangle
I tried to use RegionMask but it still targets the entire image

**Reply by jueqing1015** [reply]
@dlemstra 
May I ask if the method I said is achievable?
Specify an area on the image and perform the BoundingBox operation

**Reply by dlemstra** [reply]
You will need to create that yourself. You can use your program to draw an area on the image and then you will need to remember the coordinates. And then with these coordinates you can use `.Clone(IMagickGeometry)` to clone an area of the image.

**Reply by jueqing1015** [reply]
thank you very much
